<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupEditor</class>
 <widget class="QDialog" name="GroupEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(116, 82, 226);</string>
  </property>
  <widget class="QLineEdit" name="Attendancerate">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>568</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 255, 192);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ViceDelegateLineEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 255, 192);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="DelegateLineEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 255, 192);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Malepercentage">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>452</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 255, 192);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="FemalePercentage">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>510</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 255, 192);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>222</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Vice-Delegate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>129</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Delegate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>452</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Male percentage %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>568</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Attendance Rate %</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumberofStudents">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>394</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 255, 192);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>394</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of students</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DelegateComboBox">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>160</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Savebutton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>640</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ViceDelegateComboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Female percentage %</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Disqualifybuttondelegate">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>disqualify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DisqualifybuttonSubdelegate">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>disqualify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/reverse1_resized.jpeg);
background-repeat: no-repeat;
background-position: center;
background-size: 1px 3px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteGroupButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/removeitem_resized.jpeg);
background-position: center;
background-repitition: no-repitition;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="GroupNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 255, 192);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeGroupButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Group Number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
